Tests/Model/ModelManagerTest.php

```diff
@@ -296,3 +362,35 @@ private function getMetadata($class, $isVersioned)
         return $metadata;
     }
 }
+
+class Uuid
```

Comment:
This class should be on a separated file under `Fixtures` folder.

And if possible, use a better name that show it's a class only for test.